s3-uploads.php

```diff
@@ -14,6 +14,14 @@
 
 add_action( 'plugins_loaded', 's3_uploads_init' );
 
+// Add filters to "wrap" the wp_privacy_personal_data_export_file function call as we need to
+// switch out the personal_data directory to a local temp folder, and then upload after it's
+// complete, as Core tries to write directly to the ZipArchive which won't work with the
+// S3 streamWrapper.
+add_action( 'wp_privacy_personal_data_export_file', 's3_uploads_before_export_personal_data', 9 );
+add_action( 'wp_privacy_personal_data_export_file', 's3_uploads_after_export_personal_data', 11 );
+add_action( 'wp_privacy_personal_data_export_file_created', 's3_uplodas_move_temp_personal_data_to_s3', 1000 );
```

Comment:
Can we hook these in `s3_uploads_init()` instead? Right now, they'll be hooked in even if S3 Uploads isn't active (e.g. missing bucket constant).